app/population/controllers/edit/details.js

```diff
@@ -9,14 +9,7 @@ class DetailsController extends FormWizardController {
     this.use(this.setBreadcrumbs)
     this.use(this.setCancelUrl)
     this.use(this.setPageTitle)
-  }
-
-  setInitialValues(req, res, next) {
```

Comment:
So are the initial values no longer added? Would this not effect the "edit" state?

Reply:
This has been extracted out into separate controllers now.

On entry to the form, we check if the `req.population` exists. If it does, then we call `LoadPopulationController` and use the population data, if it doesn't we call `PopulateController` which invokes the populate method to get data - both of these set the returned population onto the session - this is then used by the `DetailsController` to display.

Reply:
Yes, I did see that later on. Okay, if it works then it's all good. I was a little confused at first.